src/main/java/sirius/kernel/xml/XMLCall.java

```diff
@@ -134,16 +135,10 @@ public XMLStructuredOutput getOutput() throws IOException {
      * @throws IOException in case of an IO error while receiving the result
      */
     public XMLStructuredInput getInput() throws IOException {
-        return new XMLStructuredInput(outcall.getResponseBody(), namespaceContext);
-    }
-
-    /**
-     * Provides access to the raw answer of the call.
-     *
-     * @return the result of the call
-     * @throws IOException in case of an IO error while receiving the result
-     */
-    public XMLStructuredInput getRawInput() throws IOException {
-        return new XMLStructuredInput(outcall.getResponse().body(), namespaceContext);
+        if (!outcall.isErroneous() || outcall.getHeaderField("content-type").toLowerCase().contains("xml")) {
```

Comment:
`getHeaderField` is nullable! Maybe check if the header is present first, then check if it contains `xml`